src/ui/services/networkService.ts

```diff
@@ -158,26 +220,23 @@ export class NetworkService extends EventEmitter {
             });
 
             const [playerlistAdd, playerlistUpdate, playerlistRemove] = this.buildListUpdater("playerlist", this.currentLobby.players, {
-                id: "id",
-                name: "name",
+                id: "id",                
+                idServer: "idServer",
+                username: "username",
                 team: ["teamId", id => this.currentLobby.teams.filter(x => x.id === id)[0]],
-                champion: ["championId", id => StaticService.champions.filter(x => x.id === id)[0]],
-                skinIndex: "skinIndex",
-                spellOne: ["spell1id", id => StaticService.summonerSpells.filter(x => x.id === id)[0]],
-                spellTwo: ["spell2id", id => StaticService.summonerSpells.filter(x => x.id === id)[0]]
+                isHost: "isHost"
             });
-            function startGame(gameServerPort){
+            function startGame(data){
                 //Start the game with the port
                 console.log("Starting LoL...")
-                var configContent = readConfigData();
                 var args = [
                     "8394",
                     "LoLLauncher.exe",
                     "",
-                    "127.0.0.1 " + gameServerPort + " 17BLOhi6KZsTtldTsizvHg== " + playerId
+                    "127.0.0.1 " + data.gameServerPort + " 17BLOhi6KZsTtldTsizvHg== " + data.playerId
                 ];
-                execFile.execFile(configContent.pathToLolExe, 
-                args, {cwd: configContent.pathToLolFolder, maxBuffer: 1024 * 90000},
+                execFile(localStorage.getItem("path") + "/League of Legends.exe", 
+                args, {cwd: localStorage.getItem("path"), maxBuffer: 1024 * 90000},
```

Comment:
Either leave this on a single line or indent it properly, this is one statement that looks like two.

Reply:
d